fossa-deps.yml

```diff
@@ -0,0 +1,6 @@
+custom-dependencies:
+  - name: zstd-jni native (Copyright (c) 2016-present, Facebook, Inc. All rights reserved.)
```

Comment:
Let's be explicit about `zstd-jni` native libraries using a slightly different license (BSD 3-Clause) than `zstd-jni` bindings (BSD 2-Clause).